### ISSUE TITLE
Result about MSVD

### ISSUE THREAD

**Issue by shufangxun** [m-bain/frozen-in-time#53]
Hi, Bain
       I found the experiment setting about MSVD result is not clear, meanwhile [MCQ](https://github.com/TencentARC/MCQ/issues/2) claims a different version. so I want to ask the result and setting about MSVD.

**Comment by m-bain** [comment]
Hi, sorry I was looking over old code. I treated them separately. Every caption->video retrieval pair counts towards the metric.

I avoid the nuanced setting of video->multi-caption retrieval by only reporting t2v!

Honestly there should be some standardised code evaluate all these benchmarks (i think https://github.com/albanie/collaborative-experts does this well but a tad hard to follow)

**Comment by shufangxun** [comment]
Thank you for your reply ! By the way,  whether the result about MSVD under the setting of finetune or zero-shot?
